src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -115,11 +115,11 @@ public static function getStyleDefinition($name)
             self::$styles = self::initStyles();
         }
 
-        if (!self::$styles[$name]) {
-            throw new InvalidArgumentException(sprintf('Style "%s" is not defined.', $name));
+        if (isset(self::$styles[$name])) {
+            return self::$styles[$name];
```

Comment:
but I don't see any change on the discussed line/code block, did you forget to push?